lms/djangoapps/course_api/blocks/transformers/milestones.py

```diff
@@ -85,11 +85,3 @@ def has_pending_milestones_for_user(block_key, usage_info):
             'requires',
             usage_info.user.id
         ))
-
-    @staticmethod
-    def has_staff_access_to_course(block_key, usage_info):
```

Comment:
@nasthagiri @efischer19 just as a heads-up since it won't show up in the full diff, I deleted this function. I realized I had written it but wasn't calling it. I added unit tests for staff access, and they passed without this call, so it seems that the checks for milestones and proctored exams have staff checks built in already.